src/frontend-samples/viewer-only-2d-sample/ViewerOnly2dUI.tsx

```diff
@@ -7,19 +7,35 @@ import React from "react";
 import { Viewer } from "@bentley/itwin-viewer-react";
 import { ControlsWidget } from "./ViewerOnly2dWidget";
 import { IModelConnection, ViewState } from "@bentley/imodeljs-frontend";
+import ViewerOnly2dApp from "./ViewerOnly2dApp";
+import { ModelProps } from "@bentley/imodeljs-common";
+export interface TwoDState {
+  drawingElements: JSX.Element[];
+  sheetElements: JSX.Element[];
+  sheets: ModelProps[];
+  drawings: ModelProps[];
```

Comment:
It seems redundant for the state to have both the elements and the modelprops.  I suggest only keeping the modelprops here and generating the jsx.elements in the widget as needed.  Those are only used for building the Select in the widget.

Also, I don't like the name 'TwoDState'.  This is part of the state so it doesn't need the word 'state'.  How about 'ModelLists'?